### PR TITLE
Odo pr review process

### PR DESCRIPTION
**Pull request by kadel**
describe our PR review process in develoment.md

### CODE REVIEW

**Comment by mohammedzee1000** [comment]
oops
/lgtm
/meow

File: docs/development.md [hunk 1]
```diff
@@ -90,12 +90,23 @@ Terminology we use:
 
 ### Reviewing a pull request
 
+#### The Pull Request review process
+
+1. The author submits a PR
+2. The @openshift-ci-robot automatically suggests an approver and requests 2 reviews from reviewers based on OWNERS files
+3. Reviewer goes and does the review.  
+    Once reviewer is ok with the changes he posts `/lgtm` as a comment to the PR. This will apply lgtm label.
```
Comment (cdrage) [inline docs/development.md]: `apply the lgtm label`

File: docs/development.md [hunk 2]
```diff
@@ -90,12 +90,23 @@ Terminology we use:
...
+4. Approver goes and does the review.  
+    Once the review is done and the approver accepts the changes and pots `/approve` as a comment to the PR. This will apply approve label.
```
Comment (cdrage) [inline docs/development.md]: 81vVSJ676yL _SY450_
 pots?

I think you mean posts :)

File: docs/development.md [hunk 3]
```diff
@@ -90,12 +90,23 @@ Terminology we use:
...
+5. Once PR has lgtm and approve labels and required test are passing bot will automatically merge PR.
```
Comment (cdrage) [inline docs/development.md]: `and required tests are passing`

File: docs/development.md (hunk 2 above)
Comment (kadel) [reply docs/development.md]: https://giphy.com/gifs/drama-cat-EX6wKXVM9IOlO

**Comment by cdrage** [comment]
Since I'm the approver...

/approve

**Comment by kadel** [comment]
ping @mohammedzee1000

**Pull request merged**